model/Security/Properties/vexVersion.md

```diff
@@ -1,17 +1,19 @@
 SPDX-License-Identifier: Community-Spec-1.0
 
-# vexVersion
+# vexMinRequirementsVersion
 
 ## Summary
 
-Specifies the version of the VEX document.
+Specifies the minimum requirements of data elements version that the VEX document adheres to.
```

Comment:
This change is not correct, but note that `vexVersion` defines the VEX statement version (not the document). It is used in the vex relationships.  This is the version detailed in 2.4.2 of the vex Minimum Requirements linked by Jeff above. We named it `vexVersion` (as opposed to just `version`) because there was a name clash.

Essentially, the Minimum Requirements doc states the fields all vex implementations need to define. It is a spec of specs but the source of truth is the SPDX spec itself, not the CISA document. In other words, this is a field defined by SPDX not by the Minimum Reqs.

Reply:
To clarify, we were relying on another field to serve as the document version and the summary of `vexVersion` should read "Specifies the version of a VEX statement."